### PR TITLE
Modules animate

### PR DESCRIPTION
**Pull request by mirisuzanne**
# REMEMBER: Attach this PR to the Trello card

### CODE REVIEW

**Comment by stacyk** [comment]
[I want Accoutrement on Sass Modules](https://trello.com/c/9tyINy2o/23-i-want-accoutrement-on-sass-modules)

File: sass/animate/_times.scss [hunk 1]
```diff
@@ -82,5 +88,5 @@ $times: () !default;
 @mixin add-times(
   $maps...
 ) {
-  $times: _a_map-multimerge($times, $maps...) !global;
-}
+  $times: utils.multi-merge($times, $maps...) !global;
+}
```
Comment (mirisuzanne) [inline sass/animate/_times.scss:92]: Can you set up your text editor to keep an end-of-file newline on save? I think that's a setting in vscode.

**Review by dvdherron — APPROVED** [review]

File: sass/animate/_easing.scss [hunk 2]
```diff
@@ -104,11 +110,11 @@ $_a_EASING: (
   $do: null,
   $source: $easing
 ) {
-  $source: map-merge($_a_EASING, $source);
-  $name: get-token($source, $name, $do);
+  $source: map.merge($EASING, $source);
+  $name: tokens.get($source, $name, $do);
   @return if((type-of($name) == 'string'), unquote($name), $name);
```
Comment (stacyk) [inline sass/animate/_easing.scss:115]: Wouldn't this line need a meta.type-of and string.unquote namespace?

File: sass/color/_index.scss [hunk 3]
```diff
@@ -102,12 +102,12 @@
...
-///   @include tools.contrasted('link');
-///   @include tools.contrasted(rebeccapurple);
+///   @include contrasted('link');
+///   @include contrasted(rebeccapurple);
 ///
 ///   // or return a contrasting color from a set of options
-///   border-color: tools.contrast('background', white, rebeccapurple);
-///   border-color: tools.contrast(maroon, white, rebeccapurple);
+///   border-color: contrast('background', white, rebeccapurple);
+///   border-color: contrast(maroon, white, rebeccapurple);
```
Comment (stacyk) [inline sass/color/_index.scss:110]: should the `tools.` namespace be removed here in these examples?

**Review by stacyk — COMMENTED** [review]
@mirisuzanne We're so close, I left some comments/questions, but not much else here to note. 🎉

File: sass/color/_index.scss (hunk 3 above)
Comment (mirisuzanne) [reply sass/color/_index.scss:110]: hmm, probably? I bet we haven't been consistent on that.

- we need the `tools` prefix on actual `@example` code, in order for it to compile
- Probably we should leave it off in the prose examples?

We can catch mistakes on that when we're doing a final review of the docs, but I think this is right.

**Comment by mirisuzanne** [comment]
@stacyk If the changes here look good to you, go ahead and merge the PR :)

File: sass/color/_index.scss (hunk 3 above)
Comment (stacyk) [reply sass/color/_index.scss:110]: Good point. Yeah, Im not sure we have been consistent... but I think mostly we've used it on the `@examples` but I didn't check close on these snippets to see if we added it or not.

**Review by stacyk — APPROVED** [review]

**Pull request merged**